### PR TITLE
Mes 4860 assertion catbe

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/test-report/cat-home-test/test-report.cat-home-test.page.module.ts [hunk 1]
```diff
@@ -5,7 +5,8 @@ import { IonicPageModule } from 'ionic-angular';
 import { testReportReducer } from '../test-report.reducer';
 import { TestReportAnalyticsEffects } from '../test-report.analytics.effects';
 import { TestReportEffects } from '../test-report.effects';
-import { ReverseDiagramModalComponentsModule } from '../components/reverse-diagram-modal/reverse-diagram-modal.components.module';
+import { ReverseDiagramModalComponentsModule }
```
Comment (jono-basford-bjss) [inline src/pages/test-report/cat-home-test/test-report.cat-home-test.page.module.ts:8]: this shouldnt be in? have you rebased correctly

File: src/pages/test-report/cat-home-test/test-report.cat-home-test.page.ts [hunk 2]
```diff
@@ -44,7 +44,8 @@ import { getCandidate } from '../../../modules/tests/journal-data/cat-home/candi
 import { TestDataByCategoryProvider } from '../../../providers/test-data-by-category/test-data-by-category';
 // TODO Implement Home Test selector
 import { hasManoeuvreBeenCompletedCatC } from '../../../modules/tests/test-data/cat-c/test-data.cat-c.selector';
-import { getTestRequirementsCatHome } from '../../../modules/tests/test-data/cat-home/test-requirements/test-requirements.cat-home.reducer';
+import { getTestRequirementsCatHome }
+from '../../../modules/tests/test-data/cat-home/test-requirements/test-requirements.cat-home.reducer';
```
Comment (jono-basford-bjss) [inline src/pages/test-report/cat-home-test/test-report.cat-home-test.page.ts:48]: his shouldnt be in? have you rebased correctly

**Review by jono-basford-bjss — CHANGES_REQUESTED** [review]

File: src/pages/test-report/cat-home-test/test-report.cat-home-test.page.ts (hunk 2 above)
Comment (naqvis5) [reply src/pages/test-report/cat-home-test/test-report.cat-home-test.page.ts:48]: resolved conflict

File: mock/generate-local-journal.ts [hunk 3]
```diff
@@ -189,6 +189,57 @@ const localJournal: ExaminerWorkSchedule = {
...
+          extendedTest: false,
+          progressiveAccess: false,
+          testCategory: 'ADI2',
+          vehicleGearbox: 'Manual',
+          specialNeedsCode: 'NONE',
+          welshTest: false,
+        },
+        candidate: {
+          candidateAddress: {
+            addressLine1: '999 Letsby Avenue',
```
Comment (jono-basford-bjss) [inline mock/generate-local-journal.ts:208]: You havent done the rebase correctly

**Review by jono-basford-bjss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**